### PR TITLE
Update Chinese Simplified (zh) Essentials, Exploring, Use Ethereum, Use Cases and Staking buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update Chinese Simplified (zh) 

- Essentials
- Exploring
- Use Ethereum
- Use Cases
- Staking buckets

### CODE REVIEW

File: src/content/translations/zh-tw/nft/index.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
+---
+title: 非同質化代幣 (NFT)
+description: 以太坊生態系非同質化代幣概要
+lang: zh-tw
```
Comment (minimalsm) [inline src/content/translations/zh-tw/nft/index.md:4]: Oops

File: src/content/translations/zh-tw/nft/index.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+template: use-cases
+emoji: ":frame_with_picture:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/infrastructure_transparent.png
```
Comment (minimalsm) [inline src/content/translations/zh-tw/nft/index.md:9]: @pettinarip for English pages ../../assets/~ is the correct path, but we require two more levels (i.e. ../../../../assets/~) for files on translated pages. 

Didn't we implement a fix for image paths in the frontmatter? Or was that only for images within markdown?

File: src/content/translations/zh/social-networks/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+title: 去中心化社交网络
+description: 以太坊去中心化社交网络概览
+lang: zh
+template: use-cases
+emoji: ":mega:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/ethereum-learn.png
```
Comment (minimalsm) [inline src/content/translations/zh/social-networks/index.md:9]:
```suggestion
image: ../../../../assets/ethereum-learn.png
```

File: src/content/translations/zh/decentralized-identity/index.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
+---
+title: 去中心化身份
+description: 什么是去中心化身份，它为什么很重要？
+lang: zh
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth-gif-cat.png
```
Comment (minimalsm) [inline src/content/translations/zh/decentralized-identity/index.md:9]:
```suggestion
image: ../../../../assets/eth-gif-cat.png
```

File: src/content/translations/zh-tw/nft/index.md (hunk 2 above)
Comment (pettinarip) [reply src/content/translations/zh-tw/nft/index.md:9]: No, I didn't do that in the end. I didn't know how to do it on those images so I leave it. But I guess it is something we can investigate further if it is worth it.

**Review by minimalsm — COMMENTED** [review]
After this builds we should double-check hero images for:

- social networks
- decentralized identity

If those are fine then LGTM

File: src/content/translations/zh-tw/nft/index.md (hunk 1 above)
Comment (minimalsm) [reply src/content/translations/zh-tw/nft/index.md:4]: This shouldn't be included in this PR, but it's not a blocker.

**Review by corwintines — APPROVED** [review]

**Pull request merged**